README.md

```diff
@@ -0,0 +1,47 @@
+# N8N-Getting-Started: N8N
+This example shows how to run a local **N8N service** using docker-compose.
+
+## Prerequisites
+
+## Instructions
+
+### Step 1. Create the environment file
+Copy the file `.env-n8n.example` renaming it to `.env-n8n` within the project's root folder.
+
+### Step 2. Start the service
+Start the docker-compose service by running the following command:
+
+```bash
+docker-compose up
```

Comment:
Does it make sense to add the `--build` flag since we are building from a local Dockerfile?

Reply:
I know it will build if no image already exists, but seems like it's more explicit and will be better to spell it out in case the user is not very familiar with Docker.

Reply:
Also, for the folks less aquatinted with Docker, it's good to spell out that you must be in the same dir as the docker-compose file for these commands to run.